### PR TITLE
[docs sprint] Updates docs for using synthesizers

### PR DESCRIPTION
**Pull request by ajar98**

### CODE REVIEW

File: docs/open-source/using-synthesizers.mdx [hunk 1]
```diff
@@ -46,6 +46,43 @@ server = InboundCallServer(
 In this example, the `ElevenLabsSynthesizerConfig.from_telephone_output_device()` method is used to create a configuration object for the Eleven Labs synthesizer.
 The method hardcodes some values like the `sampling_rate` and `audio_encoding` for compatibility with telephone output devices.
 
+#### ElevenLabs Input Streaming
+
+You can try out our experimental implementation of ElevenLabs' [input streaming API](https://elevenlabs.io/docs/api-reference/websockets) by passing in `experimental_websocket=True` into the config and using the `ElevenLabsWSSynthesizer`, like:
+
+```
```
Comment (adnaans) [inline docs/open-source/using-synthesizers.mdx]:
```suggestion
```python
```

File: docs/open-source/using-synthesizers.mdx [hunk 2]
```diff
@@ -46,6 +46,43 @@ server = InboundCallServer(
...
+    experimental_websocket=True
+))
+...
+```
+
+#### Play.ht v2
+
+We now support Play.ht's new [gRPC streaming API](https://docs.play.ht/reference/python-sdk-audio-streaming), which runs much faster than their HTTP API and is designed for realtime communication.
+
+```
```
Comment (adnaans) [inline docs/open-source/using-synthesizers.mdx]:
```suggestion
```python
```

File: docs/open-source/using-synthesizers.mdx [hunk 3]
```diff
@@ -46,6 +46,43 @@ server = InboundCallServer(
...
+...
+synthesizer_config=PlayHtSynthesizerConfig.from_telephone_output_device(
+    api_key=os.getenv("PLAY_HT_API_KEY"),
+    user_id=os.getenv("PLAY_HT_USER_ID"),
```
Comment (adnaans) [inline docs/open-source/using-synthesizers.mdx:81]: we need to add `version='2'` right?

**Review by adnaans — CHANGES_REQUESTED** [review]

File: docs/open-source/using-synthesizers.mdx (hunk 3 above)
Comment (ajar98) [reply docs/open-source/using-synthesizers.mdx:81]: it's defaulted

**Review by adnaans — APPROVED** [review]

**Pull request merged**